src/locales/en.json

```diff
@@ -344,8 +340,14 @@
     "pattern": "The value must match a specific pattern: %{pattern}"
   },
 
+  "update": {
+    "title": "An update is available for this service.",
+    "regular": "Perform this update from the service page to keep fetching your data and to have the latest features:",
+    "blocking": "Update it to keep fetching your data and to have the latest features:",
```

Comment:
No ':' at the end. I would make it clearer that the update is necessary to keep updating the data.

This update is *necessary* otherwise your data will not be updated. "latest features" is vague, I would remove it.

Reply:
De ce que j'avais compris, on était supposé avoir un message relativement générique d'où le fait que ce soit vague, mais je suis partante pour l'enlever :)